### PR TITLE
Create thedev.json

### PR DESCRIPTION
**Pull request by Mohamad-Mortada**
Added the domain: thedev

### CODE REVIEW

File: domains/thedev.json
Comment (WilliamDavidHarrison) [inline domains/thedev.json:8]: Needs to be an array.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/thedev.json
Comment (WilliamDavidHarrison) [inline domains/thedev.json:8]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/thedev.json
Comment (thedev132) [reply domains/thedev.json:8]: its an Nginx web server. I confirm its the IP of my server.

Comment (WilliamDavidHarrison) [reply domains/thedev.json:8]: @thedev132 What port is your website hosted on?

Comment (thedev132) [reply domains/thedev.json:8]: You can access my website via its IPv6 address. http://[2603:c024:c008:1bff:133:7a14:f85e:2e28]/fullscreen.html

Comment (thedev132) [reply domains/thedev.json:8]: It is hosted on port 80 but non accessible from IPv4

**Review by wdhdev — APPROVED** [review]
Accessible via [http://[2603:c024:c008:1bff:133:7a14:f85e:2e28]/fullscreen.html](http://[2603:c024:c008:1bff:133:7a14:f85e:2e28]/fullscreen.html)

**Comment by thedev132** [comment]
Why hasn't my pull request been merged right now? You confirmed it is accessible do I have to do something else?

**Comment by WilliamDavidHarrison** [comment]
> Why hasn't my pull request been merged right now? You confirmed it is accessible do I have to do something else?

I cannot merge PR's as I'm only a helper, please wait for a maintainer to merge it.

**Comment by thedev132** [comment]
Oh thanks for helping me! I hope a reviewer can merge my pull request soon :)

**Comment by thedev132** [comment]
Hey, It has been a few weeks and my pull request hasnt been merged yet. Is anyone gonna review my pull request?

**Pull request closed without merging**
